native-libs/deps/recipes/ndk/ndk.recipe

```diff
@@ -7,7 +7,7 @@ version="r15c"
 case $(uname -sm) in
 "Linux x86_64")
     system=linux-x86_64
-    source="https://dl.google.com/android/repository/android-ndk-$version-$system.zip#0600157c4ddf50ec15b8a037cfc474143f718fd0" ;;
+    source="https://dl.google.com/android/repository/android-ndk-$version-$system.zip#0bf02d4e8b85fd770fd7b9b2cdec57f9441f27a2" ;;
```

Comment:
Why the update here? how can I verify the hash

Reply:
That hash is wrong (it's hash for version 13b, we are using r15c).
It matches SHA1 sum provided by Google here: https://developer.android.com/ndk/downloads/older_releases.html#ndk-15c-downloads
```